Update docs according to yiisoft#18637 (#18655)
yiisoft/yii2: docs/guide/db-active-record.md

  Active Record directly next to it. You see there that `customer_id` is a property of `Order` and `id` is a property
  of `Customer`.
  

> >Warning: Relation name `relation` will lead to an `ArgumentCountError` due to implementation of the relations feature.

Reviewer: ```suggestion
> Warning: Relation name `relation` is reserved. When used it will produce `ArgumentCountError`.
```
